docs/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "OpenPodcast API specification"
+version = "0.1.0"
+description = "Specifications for the OpenPodcast API"
+authors = ["OpenPodcast API team"]
+
+[tool.poetry.dependencies]
+python = "^3.11"
+furo = "^2022.12.7"
+sphinx-copybutton = "^0.5.1"
+sphinx_design = "^0.3.0"
+sphinx_external_toc = "^0.3.1"
+sphinxcontrib-mermaid = "^0.8.1"
+accessible-pygments = "^0.0.3"
+sphinx = "==5.3.0"
+myst-parser = "^1.0.0"
+
+[tool.poetry.group.dev.dependencies]
+sphinx-autobuild = "^2021.3.14"
```

Comment:
Why is this a dev dependency but the rest isn't?

Reply:
@georgkrause since it's only used to run the dev server. I may be wrong about the nomenclature, but the rest are required to actually build the site, but autobuild is not.